packages/camera/camera/ios/Classes/FLTThreadSafeFlutterResult.h

```diff
@@ -0,0 +1,46 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#import <Flutter/Flutter.h>
+
+/**
+ Wrapper object that always delivers the result on the main thread.
+ */
+@interface FLTThreadSafeFlutterResult : NSObject
+
+/**
+ Initialize with a FlutterResult object.
+ @param result The FlutterResult object that the result will be given to.
+ */
+- (id _Nonnull)initWithResult:(FlutterResult _Nonnull)result;
```

Comment:
Prefer `nonnull` and `nullable` whenever possible (which should be everywhere in this file)